unyt/array.py

```diff
@@ -140,6 +141,7 @@
 
 NULL_UNIT = Unit()
 POWER_SIGN_MAPPING = {multiply: 1, divide: -1}
+STRICT = True
```

Comment:
I haven't found a single place where this is used. Looks like a leftover from a previous idea, can you clean it (or justify its presence ?)

Reply:
Indeed, corrected, thank you! I think this was a remnant from when I was trying to better understand the code and trialling different solutions